README.md

```diff
@@ -90,7 +99,32 @@ Each lion should have...
 Create a new lion instance with the name "Simba".
 
 ```rb
-# Your code goes here...
+class Animal
+	attr_accessor :animal_name
+	def initialize(animal_name)
+		@animal_name = animal_name
+	end
+	def greet
+		puts "Hi my name is #{animal_name}"
```

Comment:
This will look for a local variable (i.e., local to the method) called `animal_name`. You need to use `@animal_name` so that you can access that particular Animal instance's name.